Makefile

```diff
@@ -761,6 +818,18 @@ endif
 	$(LD) -r -T linker_scripts/audio_table_rodata.ld $(@:.o=.tmp) -o $@
 	@$(RM) $(@:.o=.tmp)
 
+$(BUILD_DIR)/src/audio/tables/soundfont_table.o: src/audio/tables/soundfont_table.c $(BUILD_DIR)/assets/audio/soundfont_table.h $(SOUNDFONT_HEADERS)
+ifneq ($(RUN_CC_CHECK),0)
+	$(CC_CHECK) $<
+endif
+	$(CC) -c $(CFLAGS) $(MIPS_VERSION) $(OPTFLAGS) -o $(@:.o=.tmp) $<
+	$(LD) -r -T linker_scripts/audio_table_rodata.ld $(@:.o=.tmp) -o $@
+	@$(RM) $(@:.o=.tmp)
+
+# Extra audiobank padding that doesn't belong to any soundfont file
+$(BUILD_DIR)/assets/audio/audiobank_padding.o:
+	echo ".section .rodata; .fill 0x20" | $(AS) $(ASFLAGS) -o $@
+
```

Comment:
could you say more about what this is? Could it be handled by aligning the next spec segment instead?

Reply:
There's a couple of symbols we care about here: each Audiobank file has it's start and length in `gSoundFontTable`, and there's also the `_AudiobankSegmentRomStart` and `_AudiobankSegmentRomEnd` symbols that are both used in dmadata.

This audiobank padding file is needed because the start+length of the last Audiobank file does not reach the end of the audiobank segment. So this can't go in the last soundfont since the length would be wrong, and we can't shrink the size of the Audiobank segment since the ROM end is used in dmadata and would be wrong.